Alien Adventure/Alien.swift

```diff
@@ -28,7 +28,7 @@ class Alien: SKSpriteNode {
         return requests[0]
     }
     var getNextRequest: UDRequest? {
-        if ++currentRequestIndex < requests.count {
+        if Int(currentRequestIndex += 1) < requests.count {
```

Comment:
I would suggest breaking this up into two lines:
`currentRequestIndex += 1`
`if currentRequestIndex < requests.count {`